hw/dv/data/fusesoc.hjson

```diff
@@ -10,7 +10,8 @@
                        "--target=sim",
                        "--build-root={build_dir}",
                        "--setup {fusesoc_core}"]
-  fusesoc_cores_root_dirs: ["--cores-root {proj_root}"]
+  fusesoc_cores_root_dirs: ["--cores-root {proj_root}",
+                            "--cores-root {TECH_LIB_PATH}/flash"]
```

Comment:
This path seems to be Google specific - this is breaking the private CI.

Reply:
Indeed, this path is a problem. It's strictly speaking not only Google specific, but specific to our technology library setup (Nuvoton uses this path as well). 

With the new flash primitive, we currently need `--cores-root {TECH_LIB_PATH}/flash` to be present IFF the `foundry` repository is present, since the ASIC version of `prim_flash` depends on a core file below that path.

I think this is something we should discuss in our design sync, since we have a similar issue with another soft-ip for OTP, and technically also for macro simulation models.

Reply:
@tjaychen for visibility